### ISSUE TITLE
Add MySpace CSS box to settings so users can apply arbitrary CSS

### ISSUE THREAD

**Issue by FreeTrade** [memberapp/memberapp.github.io#340]

**Comment by OrvilleRed** [comment]
Maybe not *arbitrary* CSS...

**Comment by FreeTrade** [comment]
Tell me more, why not arbitrary CSS?

**Comment by OrvilleRed** [comment]
I confess to knowing nothing about CSS. Just that "arbitrary" anything with respect to input that might be interpreted and executed somewhere raises a security flag in my head

**Comment by FreeTrade** [comment]
Yes, i did read a little about possible attack vectors with CSS. They seem a little limited, and we can place a warning around the box. It gives the member a lot of control so i think it outweighs possible security problems.

**Comment by FreeTrade** [comment]
@Real-FEELS would this be of any help during theme development?

**Comment by Real-FEELS** [comment]
Does not make an difference to me re: theme dev
Makes member.cash feel like myspace 🤣

**Comment by FreeTrade** [comment]
lol@myspace

i'm thinking myspace allows the page author to select the css, this would rather empower the user to select their own additional css

**Comment by Real-FEELS** [comment]
Yeah its a good feature to have and was popular in the 00's

**Comment by FreeTrade** [comment]
I like the name
